Classes/Providers/OTT/Services/PhoenixAPIDefines.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  PhoenixAPIDefines.swift
+//  Pods
+//
+//  Created by Rivka Peleg on 02/03/2017.
+//
+//
+
+import Foundation
+
+public enum AssetType: String {
```

Comment:
Please revert this enum to how it was, this enum should be visible to objc meaning it must be of Int type with computed property for the string value. you could look at the old implementation.